sys/androidmedia/gstamcvideodechybris.c

```diff
@@ -1872,8 +1874,7 @@ gst_amc_video_dec_handle_frame (GstVideoDecoder * decoder,
       switch (idx) {
         case INFO_TRY_AGAIN_LATER:
           GST_DEBUG_OBJECT (self, "Dequeueing input buffer timed out");
-          continue;             /* next try */
-          break;
+          goto dequeue_error;
```

Comment:
An alternative here would be to free the frame and return GST_FLOW_OK. This would drop one input frame, but the decoder should be able to recover. However, if after waiting for one second we are not able to grab an input frame probably something wrong is going on, so I would say it is more robust to return an error here.